src/ontology/mondo-edit.obo

```diff
@@ -209362,26 +209364,14 @@ id: MONDO:0010680
 name: X-linked Emery-Dreifuss muscular dystrophy
 def: "X-linked form of Emery-Dreifuss muscular dystrophy." [MONDO:patterns/x_linked]
 subset: ordo_etiological_subtype {source="Orphanet:98863"}
-synonym: "EDMD1" EXACT ABBREVIATION [DOID:0070246, MONDO:Lexical, OMIM:310300, Orphanet:98863]
-synonym: "Ehlers-Danlos syndrome, classic-like, 1" RELATED [OMIM:310300, OMIM:genemap2]
-synonym: "EMD1" EXACT ABBREVIATION [DOID:0070246]
 synonym: "Emerinopathy" EXACT [Orphanet:98863]
-synonym: "EMERY-Dreifuss muscular dystrophy 1, X-linked" RELATED [MONDO:Lexical, OMIM:310300]
-synonym: "Emery-Dreifuss muscular dystrophy 1, X-linked" EXACT [DOID:0070246]
 synonym: "Emery-Dreifuss muscular dystrophy, X-linked" EXACT [MONDO:patterns/x_linked]
-synonym: "Humeroperoneal neuromuscular disease" RELATED [OMIM:310300]
-synonym: "humeroperoneal neuromuscular disease" EXACT [DOID:0070246]
-synonym: "Humeroperoneal neuromuscular disease, formerly" RELATED [OMIM:310300]
 synonym: "muscular dystrophy, tardive Emery-Dreifuss type, with contractures" RELATED [GARD:0002102]
 synonym: "muscular dystrophy, tardive, Dreifuss-Emery type, with contractures" EXACT [DOID:0070246]
```

Comment:
This one should also be removed, and moved to MONDO:0100531 (or changed to "related")

Reply:
I made it related